.bingo/golangci-lint.mod

```diff
@@ -2,4 +2,4 @@ module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
 go 1.17
 
-require github.com/golangci/golangci-lint v1.45.2 // cmd/golangci-lint
```

Comment:
needs to update golangci-lint version and bingo version to work with golang v1.21